lib/tasks/attendance_csv_importer.rake

```diff
@@ -4,7 +4,9 @@ task attendance_csv_importer: :environment do
   # Pull single file from Wonderschool URL
   # Process single file through Attendance Processor
   # Archive file to S3
-  AttendanceCsvImporter.new.call
+  start_date = ENV['start_date']&.to_date || 1.year.before
```

Comment:
Can we rename these env vars to something like `wonderschool_attendance_import_start_date` so that it's clear when configuring what the start date/end date are for?